github-dark.css

```
@@ -9105,30 +9105,35 @@
   /* interactive button - bright border */
   .btn-primary:not(.compare-button):not(.btn-outline):not([disabled]):not(.disabled),
   .button.primary, .btn-theme-green, .btn-primary-mktg {
-    background: linear-gradient(#407045, #305530) !important;
-    border-color: #083 !important;
-  }
-  .button.primary:disabled, .button.primary:disabled:hover,
-  .button.primary.disabled, .button.primary.disabled:hover,
-  .btn-primary:disabled, .btn-primary:disabled:hover, .btn-primary.disabled,
-  .btn-primary.disabled:hover, .btn-theme-green:disabled,
-  .btn-theme-green:disabled:hover {
-    background: linear-gradient(#203522, #152715) !important;
+    background: linear-gradient(180deg, #170 5%, #060 100%) !important;
+    border-color: #193 !important;
+  }
+  .button.primary:disabled, .button.primary.disabled, .btn-primary:disabled,
+  .btn-primary.disabled, .button.primary:disabled:hover,
+  .button.primary.disabled:hover, .btn-primary:disabled:hover,
+  .btn-primary.disabled:hover {
+    background: linear-gradient(0deg, #030 5%, #040 100%) !important;
     border-color: #041 !important;
+    color: rgba(255, 255, 255, .3) !important;
+    opacity: .7;
   }
-  /* New additions */
-  .repo-owner-tag .tag, .gist-owner-tag .tag {
-    background: #070 !important;
-    color: #ddd !important;
+  .btn-group-squash.border-right-0 {
+    border-right: 1px solid #111 !important;
   }
   /* Green hovered */
-  .noproblems .pageheader_wrap, a.new-repo:hover {
-    background: #373 !important;
-  }
   .btn-primary:not(:disabled):not(.disabled):not(.compare-button):not(.btn-outline):hover,
   .button.primary:not(:disabled):hover, .btn-theme-green:hover,
   .btn-primary-mktg:hover {
-    background: linear-gradient(#508055, #407045) !important;
+    background: linear-gradient(180deg, #170 5%, #170 100%) !important;
```

```suggestion
    background: linear-gradient(180deg, #280 5%, #170 100%) !important;
```

Hover should not remove gradient.

---

Thats far too much, even `#180` is already too much

---

How about
```suggestion
    background: linear-gradient(180deg, #180 1%, #170 70%) !important;
```

---

lol whats with the orange

---

<s>lol whats with the orange<\s>

Never mind GH is misbehaving.